awesome_owl/static/src/card/card.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module **/
```

Comment:
Funny, it seems `#1` in the commit title turns it into a link on github... so I guess it's better to avoid it.